core/src/main/java/cucumber/runtime/formatter/PrettyFormatter.java

```diff
@@ -150,7 +151,9 @@ private void handleTestStepFinished(TestStepFinished event) {
     }
 
     private void handleWrite(WriteEvent event) {
-        out.println(event.text);
+        out.println();
+        out.println(STEP_SCENARIO_INDENT + event.text);
+        out.println();
```

Comment:
You may want to test this with multi-line strings.

Reply:
You are right, I added some changes